presto-main/src/main/java/com/facebook/presto/server/ServerConfig.java

```diff
@@ -241,4 +243,16 @@ public ServerConfig setClusterResourceGroupStateInfoExpirationDuration(Duration
         this.clusterResourceGroupStateInfoExpirationDuration = clusterResourceGroupStateInfoExpirationDuration;
         return this;
     }
+
+    public String getRestUri()
```

Comment:
Please do not add the config to `ServerConfig`.  Add it as a new `Config` instead.  See `SqlFunctionLanguageConfig`--except, instead of building this and using the value inside of a module, you'll just want to do `configBinder(binder).bindConfig(MyNewConfig.class)`